lib/aws-record-rails.rb

```diff
@@ -14,14 +14,12 @@
 
 module Aws
   module Record
-    module Rails
+    # Rails tasks for Aws::Record
+    class Railtie < ::Rails::Railtie
```

Comment:
Should the file name be changed to match the namespace?

Reply:
Disregard - just realized the gem name, heh.

Reply:
No it should be the gem name here, but yeah I think "best practice" would be a railtie file.